### DISCUSSION TITLE
Custom World Creation

### DISCUSSION THREAD

**Discussion by HzDeimos** [BG-Software-LLC/SuperiorSkyblock2#1271] [Pending Feature Requests] [answered]
Is there a way to create custom worlds that act just like the nether or the end?

I've noticed that when using the _unlock world_ command there is an option to unlock the "custom" world, but I don't know if its possible to add and configure custom worlds.

If not this should **definitely** be a feature.

**Comment by WarnDa** [comment]
Yes !
I'm trying to make the "end" world (unlocked with the end_portal) to be a NORMAL world, so it allows player to get a second island, but instead of getting useless end, he got a normal environment world, but impossible

**Reply by HzDeimos** [reply]
You can change the end schematic to achieve this. Just change the biome to plains and save the schematic, so when a player unlocks the end and goes through the portal they can have a secondary island instead of the end.

**Reply by WarnDa** [reply]
![image](https://user-images.githubusercontent.com/31826333/187996316-b02f7916-bb24-420d-bc41-2799ed9c614c.png)
I used my main schematic, in plain biome, but the sky is still end

**Reply by HzDeimos** [reply]
You need to change the biome on the config.yml

**Reply by WarnDa** [reply]
> end:
>     enabled: true
>     unlock: false
>     name: ''
>     schematic-offset: false
>     biome: PLAINS
>     dragon-fight:
>       enabled: false
>       portal-offset: 0, 0, 0

  
  Already tried, you are sure you suceed doing this ? I tried a lot of time, and only one time it worked but sky was bugged between overworld and end

**Comment by Lemvs** [comment, marked as answer]
Implemented in the stable version some time ago.

@OmerBenGera you can mark this as answered and close it.
